src/content/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer.mdx

```diff
@@ -55,17 +55,14 @@ Other things the data explorer helps you do:
 
 ## Types of data you can explore [#data-types]
 
-The **Metrics and events** explorer gives you access to these data types: 
+In the context of the **Metrics and events** data explorer, `events` is used in a general way to refer to all objects stored in the New Relic database. The data explorer lets you explore: 
```

Comment:
How about something a bit simpler like: "For the **Metrics and events** data explorer, events refers to all objects stored..."

I almost want another sentence about what "metrics" refers to here too, just for a feeling of completion.